config/manifests/bases/jaeger-operator.clusterserviceversion.yaml

```diff
@@ -15,7 +15,7 @@ metadata:
     capabilities: Deep Insights
     categories: Logging & Tracing
     certified: "false"
-    containerImage: quay.io/jaegertracing/jaeger-operator:1.56.0
+    containerImage: quay.io/jaegertracing/jaeger-operator:1.59.0
     createdAt: "2023-05-16T04:47:12Z"
```

Comment:
Curious why you capture the date.

Reply:
Which date are you referring to? `createdAt` is displayed on the openshift operator hub and the community version ([example](https://operatorhub.io/operator/jaeger)).

AFAIK it has no additional meaning. While looking at it, its a but weird, that there is no `updatedAt` filed.